OutOfSchool/OutOfSchool.WebApi/Services/BlockedProviderParentService.cs

```diff
@@ -157,4 +165,13 @@ public async Task<bool> IsBlocked(Guid parentId, Guid providerId)
     {
         return Task.FromResult<IEnumerable<string>>(new List<string>() { objectId.ToString() });
     }
+
+    private async Task<IEnumerable<BlockedProviderParent>> GetBlockedProviderParentEntities(Guid parentId, Guid providerId)
+    {
+        var currentBlock = await blockedProviderParentRepository.GetByFilter(
+            b => b.ParentId == parentId
+                 && b.ProviderId == providerId
+                 && b.DateTimeTo == null).ConfigureAwait(false);
+        return currentBlock;
+    }
```

Comment:
(Code style) This function (or, more precisely, the predicate passed as `whereExpression` to `GetByFilter()`) looks like something (specification pattern?) innate to the repository and thus could be moved there.

Reply:
I've moved it to repository. Thank you!